denis.roman/lesson2task1.rb

```diff
@@ -0,0 +1,76 @@
+class Tamagochi
+
+  ALLOWED = %w[play feed drink]
+
+  def initialize(health = 100, hunger = 0, thirst = 0)
+    @health = health
+    @hunger = hunger
+    @thirst = thirst
+  end
+
+  def start
+    loop do
+      puts 'Choose an action with your pet'
+      action = gets.chomp
+      ALLOWED.include?(action) ? send(action) : start_error
+      break if @health.eql? 0
+      status
+    end
+  end
+
+  def status
+    puts "Pet status: #{@health} health, #{@hunger} hunger and #{@thirst} thirst."
+  end
+
+  def play
+    @hunger += 20
+    @thirst += 20
+    @health -=15
+    puts "You and your pet playing. He is tired and lose some health :("
+    death_check
+  end
+
+  def feed
+    @health += 20
+    @hunger -= 50
+    puts "Your pet eats"
+    fullness_check
+  end
+
+  def drink
+    @health += 10
+    @thirst -= 50
+    puts "Your pet drinks"
+    fullness_check
+  end
+
+  private
+
+  def start_error
+    puts 'This command doesnt exist. Use only one of these: play, feed, drink.'
+  end
+
+  def death
+    @health = 0
+    @thirst = 0
+    @hunger = 0
+    puts 'Your pet has died.'
+  end
+
+  def death_check
```

Comment:
you should combine death_check death_check methods into one method, and call this method after each of action methods